apps/01PigLatin.js

```diff
@@ -7,11 +7,87 @@ prompt.start();
 
 function pigLatin(word) {
 
-    // Your code here
+  // Concept Check
+  //
+  // Please explain why we are using String.prototype.toLowerCase() here.
+  // For more info:
+  // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/toLowerCase
+  //
+  // Your Answer: We are using String.prototype.toLowerCase() to neutralize the
+  // user input and make it easier for our functions to analyze.
```

Comment:
This is true! It will make the functions easier to analyze. Why will they be easier to analyze now with toLowerCase()?